app/controllers/sessions_controller.rb

```diff
@@ -13,4 +13,11 @@ def create
     session[:user_id] = user.id
     redirect_to dashboard_index_path
   end
+
+  def destroy
+    session.delete(:user_id)
```

Comment:
Fun fact, this method will return the value of they key-value pair that is being deleted!